relay-wstring/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "relay-wstring"
```

Comment:
Please add this crate to the places where we have to list crates in `relay/src/setup.rs` and `relay/src/main.rs`.